TIL/daily/2025-03-20.md

```
@@ -0,0 +1,113 @@
+- 생성은 가능하지만, 하위 클래스의 메소드 구현은 불가능하다
+
+```dart
+abstract class Character {
+  String name;
+  
+  Character({required this.name});
+}
+
+class Hero extends Character {
+
+	Hero({required super.name});
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)